.release/ci.hcl

```diff
@@ -0,0 +1,277 @@
+schema = "1"
+
+project "consul-template" {
+  team = "consul-template"
+  slack {
+    notification_channel = "C026W707YHJ"
+  }
+  github {
+    organization = "hashicorp"
+    repository = "consul-template"
+    release_branches = ["master"]
+  }
+}
+
+event "merge" {
+  // "entrypoint" to use if build is not run automatically
+  // i.e. send "merge" complete signal to orchestrator to trigger build
+}
+
+event "build" {
+  depends = ["merge"]
+  action "build" {
+    organization = "hashicorp"
+    repository = "consul-template"
+    workflow = "build"
+  }
+}
+
+event "upload-dev" {
+  depends = ["build"]
+  action "upload-dev" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "upload-dev"
+    depends = ["build"]
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "promote-dev-docker" {
```

Comment:
I'm not sure if this event is in the correct place. It should go below the `verify` event. These workflows rely [`depend`] on each other to succeed and run in consecutive order.

Reply:
Only the project, team, notification_channel, repository and release_branches fields differ from the crt-core-helloworld example. So I'm not sure why this is this way, it's from the example.

Reply:
@sarahethompson do you know if this was left over from testing or?

Reply:
Updating the placement in line with Sarah's PR.